public_website/__tests__/home.test.tsx

```diff
@@ -32,4 +33,31 @@ describe('Home page', () => {
 
     expect(checkbox).toHaveProperty('checked', true)
   })
+
+  it('should render the page with the response of the server', async () => {
+    process.env = { ...process.env, ID_TOKEN: 'your_dummy_token' }
+
+    const { props } = await getStaticProps()
+    render(<Home {...props} />)
+
+    expect(
+      screen.queryByText(activePlaylistTagsFixtures[0]?.attributes.name ?? '')
```

Comment:
Je trouve ça très bizarre l'utilisation de ?? dans les tests..

Reply:
C'est parce que c'est potentiellement undefined
`No overload matches this call.
  Overload 1 of 2, '(id: Matcher, options?: SelectorMatcherOptions | undefined): HTMLElement | null', gave the following error.
    Argument of type 'string | undefined' is not assignable to parameter of type 'Matcher'.`